gallery/src/components/NxTabs/NxTabsModalExample.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, {useState} from 'react';
+
+import {
+  NxTabs,
+  NxTabList,
+  NxTab,
+  NxTabPanel,
+  NxModal,
+  NxFontAwesomeIcon
+} from '@sonatype/react-shared-components';
+
+import {faAngry} from '@fortawesome/free-solid-svg-icons';
+
+const NxTabsModalExample = () => {
+  const [showModal, setShowModal] = useState(false);
+  const modalCloseHandler = () => setShowModal(false);
+
+  const [activeTabId, setActiveTabId] = useState<number>(0);
+
+  function handleTabSelect(index: number) {
+    setActiveTabId(index);
+  }
+
+  return (
+    <>
+      <button onClick={() => setShowModal(true)} className="nx-btn">Open Modal</button>
+      { showModal &&
+        <NxModal id="nx-modal-simple-example" onClose={modalCloseHandler} className="nx-modal--wide">
+          <header className="nx-modal-header">
+            <h2 className="nx-h2">
+              <NxFontAwesomeIcon icon={faAngry} />
+              <span>Modal header</span>
+            </h2>
+          </header>
+          <div className="nx-modal-content">
+            <NxTabs activeTab={activeTabId} onTabSelect={handleTabSelect}>
+              <NxTabList aria-label="Simple Tabs">
+                <NxTab data-text="Tab">Tab</NxTab>
+                <NxTab data-text="Tab with longer name">Tab with longer name</NxTab>
+                <NxTab data-text="Another Tab 3">Another Tab 3</NxTab>
+                <NxTab data-text="Forth Tab">Forth Tab</NxTab>
+              </NxTabList>
+              <NxTabPanel>Tab 1</NxTabPanel>
```

Comment:
Adding larger content would probably be good.  How should the tab panel interact with the modal scrolling?

Reply:
> Adding larger content would probably be good. How should the tab panel interact with the modal scrolling?

I'll bet this isn't what we want:

<img width="1022" alt="Screen Shot 2020-09-29 at 2 50 04 PM" src="https://user-images.githubusercontent.com/1991443/94602656-29b9ea80-0263-11eb-87b3-d5c23499e2bc.png">

@dturner32 ^^

Reply:
> I'll bet this isn't what we want:

My thinking is that when an `NxTabs` is an only child of `.nx-modal-content`, it might get `flex-shrink: 1` such that it grows only to the size of the container and no higher.  Then within, it would use flexbox to manage the height of the tab panel, which would then have `overflow-y: auto`. This should institute a scrollbar within the tabe panel when necessary.

Reply:
for this one I was thinking that the tab bar would become the topmost element in the modal, an instance where the modal header would be replaced, and then within the tab panel there would be a tab panel header. This way only the content in the tab panel itself scrolls but the tab bar doesn't

Reply:
> > I'll bet this isn't what we want:
> 
> My thinking is that when an `NxTabs` is an only child of `.nx-modal-content`, it might get `flex-shrink: 1` such that it grows only to the size of the container and no higher. Then within, it would use flexbox to manage the height of the tab panel, which would then have `overflow-y: auto`. This should institute a scrollbar within the tabe panel when necessary.

Used `flex` to solve the problem here: https://github.com/sonatype/sonatype-react-shared-components/pull/153/commits/9cb5091bae572c01d746a3e788bbda830dc616af